src/intrinsics/ecma262/SymbolPrototype.js

```diff
@@ -22,8 +22,14 @@ export default function(realm: Realm, obj: ObjectValue): void {
     const target = context instanceof ObjectValue ? context.$SymbolData : context;
     if (target instanceof AbstractValue && target.getType() === SymbolValue) {
       const codeTemplate = "(A).toString()";
-      return realm.createAbstract(new TypesDomain(StringValue), ValuesDomain.topVal, [target], ([a]) =>
-        buildExpressionTemplate(codeTemplate)(realm.preludeGenerator)({ A: a })
+      let [hash, args] = hashCall(codeTemplate, target);
```

Comment:
This code block is a total copy&paste of the `toString` handling in NumberPrototype.js. That's of course not a problem introduced by this pull request, but this pull request does make the code duplication more elaborate...

Reply:
I'm trying to figure out ways to make this less bad. The changes will be rather intrusive and should be done in a series of separate pull requests.